src/GenFacades/Program.cs

```diff
@@ -314,7 +314,7 @@ private static IEnumerable<string> EnumerateDocIdsToForward(IAssembly contractAs
             // Make note that all type forwards (including nested) implement INamespaceAliasForType, so do
```

Comment:
What I thought the comment was indicating is the explanation of why to use `alias.AliasedType` instead of just `alias`. Although, I also get that perhaps we were intentionally filtering out nested forwarders for a reason?